### PR TITLE
[d3d9] Clean up fetch4 handling

### PR DESCRIPTION
**Pull request by Joshua-Ashton**
@K0bin

### CODE REVIEW

**Comment by Joshua-Ashton** [comment]
TODO: Change it in SetStateTexture

**Review by K0bin — CHANGES_REQUESTED** [review]
I like it but you didn't change the fetch4 code when binding a new texture. That's what caused the bug in Spider Man.

**Comment by Joshua-Ashton** [comment]
ye, fixed

**Review by K0bin — APPROVED** [review]

File: src/d3d9/d3d9_device.cpp [hunk 1]
```diff
@@ -3764,7 +3752,7 @@ namespace dxvk {
         m_flags.set(D3D9DeviceFlag::DirtyFFPixelShader);
     }
 
-    DWORD oldUsage = oldTexture != nullptr ? oldTexture->Desc()->Usage : 0;
+    DWORD oldUsage = oldTexture != nullptr ? oldTecd xture->Desc()->Usage : 0;
```
Comment (pchome) [inline src/d3d9/d3d9_device.cpp:3755]: this ?

Comment (Joshua-Ashton) [reply src/d3d9/d3d9_device.cpp:3755]: Typo ye

**Pull request merged**